GameServerLib/Logic/GameObjects/Spells/Spell.cs

```diff
@@ -69,7 +69,10 @@ public Spell(Champion owner, string spellName, byte slot)
             _scriptEngine = Program.ResolveDependency<CSharpScriptEngine>();
 
             //Set the game script for the spell
-            spellGameScript = _scriptEngine.CreateObject<GameScript>("Spells", spellName);
+            if (spellName != "")
```

Comment:
`!spellName.Equals("")`, or even better: `!string.IsNullOrEmpty(spellName)`

here is why: https://stackoverflow.com/a/814881